### PR TITLE
Applied correct divisors to voltage for Alpha UPSs

### PR DESCRIPTION
**Pull request by noaheroufus**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

---
This change is [Reviewable](https://reviewable.io/reviews/librenms/librenms/7377)

### CODE REVIEW

File: includes/definitions/discovery/fxm.yaml [hunk 1]
```diff
@@ -32,16 +34,17 @@ modules:
                     oid: upsOutputFrequency
                     num_oid: .1.3.6.1.4.1.7309.6.1.4.2.
                     index: 'upsOutputFrequency.{{ $index }}'
-                    divisor: 10
                     descr: Output
+                    divisor: 10
         current:
+            options:
+              divisor: 10
```
Comment (laf) [inline includes/definitions/discovery/fxm.yaml]: This doesn't change the divisor. You've just moved it to be applied to all currents detected.

File: includes/definitions/discovery/fxm.yaml [hunk 2]
```diff
@@ -32,16 +34,17 @@ modules:
                     oid: upsOutputFrequency
                     num_oid: .1.3.6.1.4.1.7309.6.1.4.2.
                     index: 'upsOutputFrequency.{{ $index }}'
-                    divisor: 10
                     descr: Output
+                    divisor: 10
```
Comment (laf) [inline includes/definitions/discovery/fxm.yaml]: This has just moved a couple of lines, any reason?

File: includes/definitions/discovery/fxm.yaml [hunk 3]
```diff
@@ -2,6 +2,8 @@ mib: mibs/alpha/Argus-Power-System-MIB
 modules:
     sensors:
         voltage:
+            options:
+              divisor: 10
```
Comment (laf) [inline includes/definitions/discovery/fxm.yaml]: So this is the only thing affecting graphs?

File: includes/definitions/discovery/fxm.yaml (hunk 2 above)
Comment (noaheroufus) [reply includes/definitions/discovery/fxm.yaml]: I just noticed the apc.yaml had the divisors after the descr line.

File: includes/definitions/discovery/fxm.yaml (hunk 3 above)
Comment (noaheroufus) [reply includes/definitions/discovery/fxm.yaml]: Yes, all three appear to need the divisor.

File: includes/definitions/discovery/fxm.yaml (hunk 2 above)
Comment (laf) [reply includes/definitions/discovery/fxm.yaml]: The order doesn't matter.

File: includes/definitions/discovery/fxm.yaml (hunk 3 above)
Comment (laf) [reply includes/definitions/discovery/fxm.yaml]: Ok but the PR states it fixes current, voltage and frequency but I can only see voltage as being the one that is actually changed.

File: includes/definitions/discovery/fxm.yaml (hunk 1 above)
Comment (noaheroufus) [reply includes/definitions/discovery/fxm.yaml]: Ignore that

File: includes/definitions/discovery/fxm.yaml (hunk 3 above)
Comment (noaheroufus) [reply includes/definitions/discovery/fxm.yaml]: Changed the title to correspond with *just* voltage being changed.

**Comment by laf** [comment]
Thanks for the fix, quick responses and congrats on your first PR :)

image

**Pull request merged**